index.html

```diff
@@ -0,0 +1,57 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Awesome Books</title>
+
+    <link rel="stylesheet" href="index.css" />
+  </head>
+  <body>
+    <header class="container header">
+      <p class="page-title">Awesome Books</p>
+      <nav>
+        <ul class="nav-links">
+          <li class="nav-link nav-link--active" data-sect="1">List</li>
```

Comment:
- [ ] I kindly suggest that you make sure your switching of pages functionality is working, as I can't switch to the `Add New` and `Contact page`. See  video below for demo

https://user-images.githubusercontent.com/52934128/177163726-0189b1a5-9303-404b-adff-2ab7d27a90b2.mp4